src/obb.cpp

```diff
@@ -42,16 +42,30 @@ class OBBPrivate : public FCL_OBB
 
 OBB::OBB()
 {
-  this->obb_.reset(new OBBPrivate);
+  obb_.reset(new OBBPrivate);
+  // Initialize the OBB to position 0, with 0 extents and identity rotation
+#if FCL_MAJOR_VERSION > 0 || FCL_MINOR_VERSION > 5
+  // FCL 0.6+ does not zero-initialize the OBB.
+  obb_->extent.setZero();
+  obb_->To.setZero();
+  obb_->axis.setIdentity();
+#else
+  // FCL 0.5 zero-initializes the OBB, so we just put the identity into orientation.
+  obb_->axis[0][0] = 1.0;
+  obb_->axis[1][1] = 1.0;
+  obb_->axis[2][2] = 1.0;
+#endif
 }
 
-OBB::OBB(const OBB& other) : OBB()
+OBB::OBB(const OBB& other)
 {
+  obb_.reset(new OBBPrivate);
   *obb_ = *other.obb_;
```

Comment:
What about using the copy constructor of OBBPrivate?
```suggestion
  obb_.reset(new OBBPrivate(*other.obb_);
```

Reply:
Thanks for the suggestion. Applied.